README.md

```diff
@@ -1 +1,113 @@
-# ToDo-LIST
\ No newline at end of file
+# ToDo-LIST
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Your live demo link in the Table of Contents is broken. Kindly fix it by adding a linked section or removing this entirely.